rs/nns/cmc/src/main.rs

```diff
@@ -2433,31 +2452,20 @@ async fn do_create_canister(
     Err(last_err.unwrap_or_else(|| "Unknown problem attempting to create a canister.".to_owned()))
 }
 
```

Comment:
```suggestion

/// If the cycles balance of this canister (i.e. Cycles Minting) is less than `cycles`, mint more
/// so that the balance of this canister is at least `cycles`.
```

You could also mention something about limiting, but this is secondary.